specs/validator/beacon_node_oapi.yaml

```diff
@@ -0,0 +1,609 @@
+        503:
+          $ref: '#/components/responses/CurrentlySyncing'
+
+components:
+  schemas:
+    pubkey:
+      type: string
+      format: byte
+      pattern: "^0x[a-fA-F0-9]{96}$"
+      description: "The validator's BLS public key, uniquely identifying them. _48-bytes, hex encoded with 0x prefix, case insensitive._"
+      example: "0x1b66ac1fb663c9bc59509846d6ec05345bd908eda73e670af888da41af171505cc411d61252fb6cb3fa0017b679f8bb2305b26a285fa2737f175668d0dff91cc"
+    version:
+      type: string
+      description: "A string which uniquely identifies the client implementation and its version; similar to [HTTP User-Agent](https://tools.ietf.org/html/rfc7231#section-5.5.3)."
+      example: "Lighthouse / v0.1.5 (Linux x86_64)"
+    genesis_time:
+      type: integer
+      format: uint64
+      description: "The genesis_time configured for the beacon node, which is the time the Eth1.0 validator deposit smart contract has enough ETH staked (i.e. Eth2.0 begins)."
+      example: 1557716289
+    ValidatorDuty:
+      type: object
+      properties:
+        validator_pubkey:
+          $ref: '#/components/schemas/pubkey'
+        committee_index:
+          type: integer
+          format: uint64
+          description: "The index of the validator in the committee."
+        attestation_slot:
+          type: integer
+          format: uint64
+          description: "The slot at which the validator must attest."
+        attestation_shard:
+          type: integer
+          format: uint64
+          description: "The shard in which the validator must attest."
+        block_proposal_slot:
+          type: integer
+          format: uint64
+          nullable: true
+          description: "The slot in which a validator must propose a block, or `null` if block production is not required."
+    SyncingStatus:
+      type: object
+      nullable: true
+      properties:
+        starting_block:
+          type: integer
+          format: uint64
+          description: "The block at which syncing started (will only be reset after the sync reached its head)"
+        current_block:
+          type: integer
+          format: uint64
+          description: "The current highest block sync'd by the beacon node."
+        highest_block:
+          type: integer
+          format: uint64
+          description: "The estimated highest block, or current target block number."
+
+    BeaconBlock:
+      description: "The [`BeaconBlock`](https://github.com/ethereum/eth2.0-specs/blob/dev/specs/core/0_beacon-chain.md#beaconblock) object from the Eth2.0 spec."
+      allOf:
+        - $ref: '#/components/schemas/BeaconBlockCommon'
+        - type: object
+          properties:
+            body:
+              $ref: '#/components/schemas/BeaconBlockBody'
+    BeaconBlockHeader:
+      description: "The [`BeaconBlockHeader`](https://github.com/ethereum/eth2.0-specs/blob/dev/specs/core/0_beacon-chain.md#beaconblockheader) object from the Eth2.0 spec."
```

Comment:
I agree, though I am concerned about this since I think the most usable version of this document is when it's listed on SwaggerHub (or some swagger viewer). The links won't be interpreted correctly if they are relative in this case.
Maybe there is a better absolute url that I can use, rather than the `dev` branch?
Otherwise, if you think they should be relative links regardless, happy to update them.

Reply:
Interesting.. `master` is generally more stable and contains the releases so that would be my preference here. I'll change it